code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -213,17 +211,16 @@ descPairs IncludeUnits =
   map (\x -> (P $ eqSymb x, Flat $ phrase x +:+ sParen (toSentenceUnitless x), Nothing))
   -- FIXME: Need a Units map for looking up units from variables
 
--- | Defines 'Field's as 'String's.
-instance Show Field where
-  show Label             = "Label"
-  show Symbol            = "Symbol"
-  show Units             = "Units"
-  show RefBy             = "RefBy"
-  show Source            = "Source"
-  show Input             = "Input"
-  show Output            = "Output"
-  show InConstraints     = "Input Constraints"
-  show OutConstraints    = "Output Constraints"
-  show DefiningEquation  = "Equation"
-  show (Description _ _) = "Description"
-  show Notes             = "Notes"
+getSent :: Field -> Sentence
+getSent Label             = S "Label"
+getSent Symbol            = S "Symbol"
+getSent Units             = S "Units"
+getSent RefBy             = S "RefBy"
+getSent Source            = S "Source"
+getSent Input             = S "Input"
+getSent Output            = S "Output"
+getSent InConstraints     = S "Input Constraints"
+getSent OutConstraints    = S "Output Constraints"
+getSent DefiningEquation  = S "Equation"
+getSent (Description _ _) = S "Description"
+getSent Notes             = S "Notes" 
```

Comment:
One of the next steps here would be passing these sentences from our previously defined objects (e.g., `atStart output_` or something similar) instead of hardcoding them like this (and add a trailing new line to this file - it doesn't look like there is one but I could be wrong 😅).

Reply:
just had some issues with the generation of the JSON and tex files I have fixed those.